tests/sensors/test_timeout_sensor.py

```diff
@@ -72,6 +74,7 @@ def setUp(self):
         }
         self.dag = DAG(TEST_DAG_ID, default_args=args)
 
+    @pytest.mark.quarantined
```

Comment:
Not end of the world, but I would have expected these to appear in #8498, not here?

Reply:
I thought about it. The problem is that if we do that in #8498 we have no CI job to run those quarantined tests - they are skipped by default unless we add --include-quarantined. So i decided to introduce it in two steps:

1) include the quarantined tests marker and fix the --system --integration flags to be repeatable (so this is a kind of "re-organizing") PR.

2) add  CI job update to include "quarantine" job (among other CI changes). 

If you think this might be better if we further split, I am quite OK to split it further. For example:

Commit 1. Re-organize --system and --integration 
Commit 2. Add cI.yml modificaitons withtout having "quarantine" jon
Commit 3. Add "quarantined" job + "quarantined" marker + mark all the tests as quarantined.

Do you think it will be better this way @ashb ?

I am happy to do it and it should be rather easy - the new IntelliJ commit feature is great when it comes to splitting such commits, especially tha it integrates well with all the static/checks, pre-commits, and all the usual IDE features. The best thing about it is that it is non-modal, so you can work in your IDE as usual and do the usual search/replace/verify/find usages etc. while splitting the commit.

Reply:
Yeah. I split it slightly differently now. --integration and --system flag changes went to the #8496 and all quarantine-related changes are moved to #8511